README.md

```diff
@@ -23,7 +23,7 @@ To get started, simply include `main.tf` into your terraform codebase:
 ```hcl
 module "landing_zone" {
   source     = "TerraHubCorp/landing-zone/aws"
-  version    = "0.0.5"
+  version    = "0.0.6"
```

Comment:
I think the `module version` should also be reflected here. Am I right?
https://github.com/TerraHubCorp/terraform-aws-landing-zone/blob/6dd1422802eefb20765a57dfcb313e653d4c13fd/main.tf#L3